WordPress/gutenberg
# Docs: add tutorial for creating a custom block editor instance

## Description

I wrote a "tutorial" (more a "walkthrough guide") on creating a custom instance of the block editor within WP Admin.

This is highly relevant as the Full Site Editing project editor starts to expand the editor beyond Post content.

This tutorial was originally published on [a standalone repo](https://github.com/getdave/standalone-block-editor), but having spoken with @youknowriad he's suggested it might be useful as a public tutorial within the Gutenberg docs.

This PR:

* ports over the content.
* rewords as required to make suitable for Gutenberg project docs.

Note that the code for the tutorial still lives in the original repo. I wonder whether this should be moved within the Gutenberg project somehow to ensure it is preserved.

## How has this been tested?

This tutorial has been reviewed by members of the Core team including @epiqueras and @youknowriad.

To test you can:

* Read through the tutorial.
* Check it has been worded in a way that is suitable for inclusion in public-facing docs.
* Check for spelling/grammar errors.
* Ensure appropriate links are in place.


## Types of changes
New feature (non-breaking change which adds functionality).

## Checklist:
